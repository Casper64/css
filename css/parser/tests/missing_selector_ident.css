. {
    color: red;
}

# {
    color: white;
}

.test: {
    color: blue
}