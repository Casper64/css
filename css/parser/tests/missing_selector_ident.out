css/parser/tests/missing_selector_ident.css:1:1: error: expecting an identifier after "."
    1 | . {
      | ~~~
    2 |     color: red;
    3 | }
css/parser/tests/missing_selector_ident.css:5:1: error: expecting an identifier after "#" in a selector
    3 | }
    4 | 
    5 | # {
      | ~~~
    6 |     color: white;
    7 | }
css/parser/tests/missing_selector_ident.css:9:6: error: expecting an identifier or ":" after ":" in a selector
    7 | }
    8 | 
    9 | .test: {
      |      ~~~
   10 |     color: blue
   11 | }